File: API/CypherKeeper/CypherKeeper.Model/tbWebsitesModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CypherKeeper.Model
{
    [Table("tbWebsites")]
    public class tbWebsitesModel
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public String Name { get; set; }
        public String Link { get; set; }
        public Guid IconId { get; set; }
        public Boolean isDeleted { get; set; } = false;
        public DateTime DeletedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
        public DateTime CreatedDate { get; set; }
    }
}

File: CypherKeeperUI/CypherKeeper/src/app/Models/tbWebsitesModel.ts
export interface tbWebsitesModel {
    id: string;
    name: string;
    link: string | null;
    iconId: string | null;
    isDeleted: boolean;
    deletedDate: string | null;
    updatedDate: string | null;
    createdDate: string;
}

export interface tbWebsitesModel_ToAdd {
    Name: string;
    Link: string | null;
    IconId: string | null;
}